.cid

```diff
@@ -0,0 +1 @@
+QmVsGxwZs4acTDfhQrNYVqccEVxyhALqJuYqgJrbbNAd1Z
```

Comment:
Why do you need to commit this file?

Reply:
so that we have the CID for the last known weak subjectivity checkpoint stored locally

Reply:
to clarify, it's so we can have it ready to paste into the lodestar CLI when we have a new CID (upon the release of a new version of lodestar)

Reply:
if we have to, for example, migrate to a new server and re-download this repo to get the scripts running, then it would be nice to have a file with the latest checkpoint.  but then again, maybe it's possible that we haven't committed a new file for a long time, so this file could give a false sense of what is the actual latest ws state.  guess it depends on what process we decide for this?